physionet-django/events/urls.py

```diff
@@ -6,6 +6,7 @@
 urlpatterns = [
     path('', views.event_home, name='event_home'),
     path('create/', views.create_event, name='create_event'),
+    path('<slug:event_slug>/manage_co_hosts/', views.manage_co_hosts, name='manage_co_hosts'),
```

Comment:
In terms of API design, wouldn't it be cleaner to have:
1. Separate endpoints for adding and removing.
or
2. Distinguishing the action based on the method (single endpoint with `PUT` and `DELETE`) handlers.


Then there is no need to define the `CohostStatus` enum at all. Having separate endpoints also makes the view logic more comprehensible since it would handle each case separately (no branching).

Reply:
I have updated the functionality to use PUT & DELETE for the changes.

Reply:
Django has inherent support for get & post requests, but not the same for put & delete, and it seems that only these requests are supported by URL testing and hence the tests for put & delete are failing.